tests/scripts/all.sh

```diff
@@ -2248,12 +2249,16 @@ component_test_psa_crypto_config_accel_all_ec_algs_use_psa () {
 
     # Build the library
     loc_accel_flags="$loc_accel_flags $( echo "$loc_accel_list" | sed 's/[^ ]* */-DMBEDTLS_PSA_ACCEL_&/g' )"
-    make CFLAGS="$ASAN_CFLAGS -Werror -I../tests/include -I../tests -I../../tests -DPSA_CRYPTO_DRIVER_TEST -DMBEDTLS_TEST_LIBTESTDRIVER1 $loc_accel_flags" LDFLAGS="-ltestdriver1 $ASAN_CFLAGS"
+    loc_symbols="-DPSA_CRYPTO_DRIVER_TEST \
+                 -DMBEDTLS_TEST_LIBTESTDRIVER1 \
+                 -DMBEDTLS_ECP_LIGHT"
```

Comment:
Why is this passed as a CFLAGS option, not in the config file?

Reply:
Because we want to keep it private (mostly [because it doesn't have well-defined semantics](https://github.com/Mbed-TLS/mbedtls/issues/6839#issuecomment-1506571727)), so it can't appear in the config file.

Passing it in the `CFLAGS` is a temporary situation, though: in the future it will be auto-enabled based on options that need it, but [we're going step by step](https://github.com/Mbed-TLS/mbedtls/issues/6839#issuecomment-1506735581).

Reply:
See https://github.com/Mbed-TLS/mbedtls/issues/7442.